### ISSUE TITLE
Implement checks for DockerStatic Dockerfiles

### ISSUE THREAD

**Issue by sxa** [adoptium/infrastructure#2307]
We should have a process, either via github actions or an equivalent of VagrantPlaybookCheck / DockerFileCheck, for verifying changes to the `DockerStatic` Dockerfiles prior to merging PRs. We recently saw issues with a parameter to `sha512sum` causing a problem that would potentially have otherwise gone unnoticed.

**Comment by sxa** [comment]
Add `good first issue` to this. It ought to be possible to produce the scripts to automate this based on the vagrantPlaybookCheck ones. Just needs a way to start up a docker container using the Dockerfiles in https://github.com/adoptium/infrastructure/tree/master/ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/DockerStatic/Dockerfiles and try and run a test on them (i.e. download a GA build and run a test similar to [testJDK.sh](https://github.com/adoptium/infrastructure/blob/master/ansible/pbTestScripts/testJDK.sh)

**Comment by zdtsw** [comment]
I can pick this issue @sxa

**Comment by zdtsw** [comment]
anything else need to implement before we can have this ticket closed? @sxa

**Comment by sxa** [comment]
I'm trying to think about the best way to implement this but the one extra thing that would be extremely useful is if we could verify that the containers are able to run tests successfully. We have https://github.com/adoptium/infrastructure/blob/master/ansible/pbTestScripts/testJDK.sh which you point at a JDK (We could use `TEST_JDK_ROOT=/usr/lib/jvm/jdk8`) that will verify that everything is installed successfully so if we could try adjusting the github action to run that script that would be useful to catch problems before we deploy the docker containers.

**Comment by zdtsw** [comment]
I am thinking either update description of this issue to include testing image or can create another issue to do the testing. @sxa

**Comment by sxa** [comment]
The description does list 'equivalent of VagrantPlaybookCheck' which does a basic test after the deploy (that's where the `testJDK.sh` comes from) so I think this issue is ok, but I'm equally happy if you'd prefer to create another issue for it.
